app/components/shared/card_component/card_component.html.slim

```diff
@@ -0,0 +1,16 @@
+dl.card-component id=id class=classes *html_attributes
+  .card-component__content
+    dt.card-component__header
+      ul.govuk-list
+        - header_items.each do |item|
+          li == render(item)
```

Comment:
You should be able to just do `li = item`

Reply:
has to bee == or it doesnt render but yeah the render method not needed